app-modules/prospect/src/Filament/Resources/ProspectResource/Pages/ManageProspectAlerts.php

```diff
@@ -126,12 +134,20 @@ public function form(Form $form): Form
                 Textarea::make('suggested_intervention')
                     ->required()
                     ->string(),
+                Select::make('status_id')
+                    ->label('Status')
+                    ->relationship('status', 'name', fn (Builder $query) => $query->orderBy('sort'))
+                    ->default(SystemAlertStatusClassification::default())
+                    ->selectablePlaceholder(false)
+                    ->required()
+                    ->visible(AlertStatusId::active()),
                 Select::make('status')
-                    ->options(AlertStatus::class)
+                    ->options(SystemAlertStatusClassification::class)
                     ->selectablePlaceholder(false)
-                    ->default(AlertStatus::default())
+                    ->default(SystemAlertStatusClassification::default())
```

Comment:
This won't work here because you changed the `default` to return the Model. So you will need to hard code the correct default enum.